src/api.ts

```diff
@@ -147,6 +145,12 @@ interface SEOData {
     webTitle: string;
 }
 
+export enum Fronts {
```

Comment:
Could this live just in `Email.tsx` as it is only used there? It's not really related to the api model itself/rest of this file.

Reply:
Makes sense, I've moved it to `Email.tsx`